lib/active_merchant/billing/gateways/epsilon_gmo_later.rb

```diff
@@ -0,0 +1,38 @@
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class EpsilonGmoLaterGateway < EpsilonBaseGateway
+      def purchase(amount, detail = {})
+        params = {
+          contract_code: self.contract_code,
+          user_id:       detail[:user_id],
+          user_name:     detail[:user_name],
+          user_mail_add: detail[:user_email],
+          item_code:     detail[:item_code],
+          item_name:     detail[:item_name],
+          order_number:  detail[:order_number],
+          st_code:       '00000-0000-00000-00000-00100-00000-00000',
```

Comment:
決済区分は以下が正しいかと思いました

```suggestion
          st_code:       '00000-0000-00000-00010-00000-00000-00000',
```

> 書式:NNNNN-NNNN-NNNNN-NNNNN-NNNNN-NNNNN-NNNNN 
(以下表の対比12345-6789-ABCDE-FGHIJ-KLMNO-PQRST-UVWXY)

「GMO後払い決済」は対応表（p.46）を見る限り `I` だったので